envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/EnvoyAssertions.kt

```diff
@@ -32,7 +33,8 @@ fun ObjectAssert<EnvoyContainer>.hasOneAccessDenialWithActionBlock(
     path: String,
     method: String,
     clientName: String,
-    clientIp: String
+    clientIp: String,
+    requestId: String = ""
```

Comment:
Why did you set default value for `requestId` but there is no for other ?

Reply:
Request id is something that is not always present so I opted for a default value in assertions so I do not change all of the callers. I'll change that.